src/middleware.ts

```diff
@@ -5,6 +5,9 @@ export const config = {
   matcher: ['/api/og/:path*'],
 };
 
+// add this to silence request logs but let's NOT send this to prod
```

Comment:
does this mean we should remove? or like make another branch that we deploy to prod without this? 🤔